lib/views/pages/organization/create_organization.dart

```diff
@@ -148,6 +148,12 @@ class _CreateOrganizationState extends State<CreateOrganization> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
+        leading: IconButton(
```

Comment:
Didi you tried it with the appBar property automaticallyImplyLeading set to true I guess this could be more optimal if this one works though you have to keep the navigator pushReplacment to navigator push.